iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/rest/IcebergTableOperations.java

```diff
@@ -101,8 +102,12 @@ public IcebergTableOperations(
   @ResponseMetered(name = "list-table", absolute = true)
   public Response listTable(
       @PathParam("prefix") String prefix, @PathParam("namespace") String namespace) {
-    return IcebergRestUtils.ok(
-        icebergCatalogWrapperManager.getOps(prefix).listTable(RESTUtil.decodeNamespace(namespace)));
+    String catalogName = IcebergRestUtils.getCatalogName(prefix);
+    Namespace icebergNS = RESTUtil.decodeNamespace(namespace);
+    LOG.info("List Iceberg tables, catalog:{}, namespace: {}", catalogName, icebergNS);
```

Comment:
Can you please add a whitespace after ":"?

Reply:
updated